src/status_im/components/drawer/view.cljs

```diff
@@ -145,10 +145,11 @@
                :uppercase? platform/android?}
          (i18n/label :t/view-all)]]])))
 
-(defn current-network []
-  [view {:style st/network-label-container}
-   [text {:style st/network-label} (i18n/label :t/current-network)]
-   [text {:style st/network-title} "Ropsten"]])
+(defview current-network []
+  (letsubs [{:keys [name]} [:get-current-account-network]]
+    [view {:style st/network-label-container}
+     [text {:style st/network-label} (i18n/label :t/current-network)]
+     [text {:style st/network-title} name]]))
```

Comment:
Do we want to shadow `name`? It's a builtin. Can we do this with `(:name m)` instead?

Reply:
+1